### PR TITLE
add graphyte to send metrics to graphite or carbon

### PR DESCRIPTION
**Pull request by wdpypere**
not that hard to do, but add some convenience so data can be string or list as well, and some basic logging.

### CODE REVIEW

File: lib/vsc/utils/netcat.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     - bytes
+     - string
+     - list of strings
+
+    timeout value is counted for the entire connection so includes
+    sending of data.
+    """
+    # convert list to string if needed
+    if isinstance(data, list):
+        data = "\n".join(data)
```
Comment (stdweird) [inline lib/vsc/utils/netcat.py:48]: this makes no sense. i would expect either some serialiser, or that it sends element per element

Comment (stdweird) [reply lib/vsc/utils/netcat.py:48]: or https://pypi.org/project/graphyte/

Comment (wdpypere) [reply lib/vsc/utils/netcat.py:48]: valid point on the list, I was only thinking about my use case. I'll remove that.

graphyte is not available on our systems and I don't think maintaining and rpm to replace 2 lines of code is a good idea.

Comment (wdpypere) [reply lib/vsc/utils/netcat.py]: just to replace the following?
```
    with socket.create_connection((host, port), timeout=timeout) as sock:
        sock.sendall(data)
```

Comment (stdweird) [reply lib/vsc/utils/netcat.py]: you are sure we will never need any of the features in graphyte?  not so convinced tbh

**Comment by wdpypere** [comment]
@stdweird this now includes graphyte

**Pull request merged**